backend_jobs/ingestion_pipeline/pipeline_lib/image_provider_interface.py

```diff
@@ -23,6 +23,7 @@
 from abc import ABC, abstractmethod
```

Comment:
nit: the 'file documentation' is more of a class documentation - can we move it to the class?

Reply:
yes, moved.